### PR TITLE
docs: add a caveat about serialization

### PR DESCRIPTION
**Pull request by marcpicaud**
In reference to https://github.com/mikro-orm/nestjs/discussions/65

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,6 +176,32 @@ export class MyService {
...
+  author: Author;
+
+}
+
+```
+
+ 
+
+
 ## Using `AsyncLocalStorage` for request context
```
Comment (B4nan) [inline README.md:205]:
```suggestion

## Using `AsyncLocalStorage` for request context
```

File: README.md [hunk 2]
```diff
@@ -176,6 +176,32 @@ export class MyService {
...
+[NestJS built-in serialization](https://docs.nestjs.com/techniques/serialization) relies on [class-transformer](https://github.com/typestack/class-transformer). Since MikroORM wraps every single entity relation in a `Reference` or a `Collection` instance (for type-safety), this will make the built-in `ClassSerializerInterceptor` blind to any wrapped relations. In other words, if you return MikroORM entities from your HTTP or WebSocket handlers, all of their relations will NOT be serialized.
+
+Luckily, MikroORM provides a [serialization API](https://mikro-orm.io/docs/serializing) which can be used in lieu of `ClassSerializerInterceptor`.
+
+```typescript
+@Entity()
+export class Book {
+
+  @Property({ hidden: true })   // --> Equivalent of class-transformer's `@Exclude`
+  hiddenField = Date.now();
```
Comment (B4nan) [inline README.md:190]:
```suggestion
  hiddenField: number = Date.now();
```

**Review by B4nan — APPROVED** [review]

**Comment by B4nan** [comment]
Thanks, can you please add it to the nestjs recipe too?

https://github.com/mikro-orm/mikro-orm/blob/master/docs/docs/usage-with-nestjs.md
https://github.com/mikro-orm/mikro-orm/blob/master/docs/versioned_docs/version-5.0/usage-with-nestjs.md

**Comment by marcpicaud** [comment]
Here you go : 
https://github.com/mikro-orm/mikro-orm/pull/2902 (docs)
https://github.com/mikro-orm/mikro-orm/pull/2903 (versioned docs)

I also submitted a PR in the NestJS website : 
https://github.com/nestjs/docs.nestjs.com/pull/2243

**Pull request merged**